benches/nexmark/generator.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2022 the DBSP contributors.
```

Comment:
This repo uses the `mod.rs` style of modules, so could you move this into `generator/` and rename it `mod.rs`? The `_generator` is also redundant in `strings_generator.rs` and `person_generator.rs` so they can be renamed to `strings.rs` and `person.rs` (maybe `people.rs`? Seems like it rolls off the tongue better)

Reply:
Ah - ok. I was just going with the newer recommendation to avoid mod.rs - happy to switch.

Reply:
Done.